test/validation/org/apache/cassandra/bridges/Bridge.java

```diff
@@ -29,8 +29,8 @@
     public abstract String readClusterLogs(String testName);
     public abstract void captureLogs(String testName);
     public abstract String[] clusterEndpoints();
-    public abstract void nodeTool(int node, String command, String arguments);
```

Comment:
An object (maybe an abstract base class) would make the best handle for a node even if it just contains a string field. If we develop more complex ways of describing a node the intermediate code passing around the handles won't change type. Can save some typing later on.